modules/odc-setting-user-preferences.adoc

```diff
@@ -13,4 +16,7 @@ You can set the default user preferences for your cluster.
 .. In the *Project* field, select a project you want to work in. The console will default to the project every time you log in.
 .. In the *Topology* field, you can set the topology view to default to the graph or list view. If not selected, the console defaults to the last view you used.
 .. In the *Create/Edit resource method* field, you can set a preference for creating or editing a resource. If both the form and YAML options are available, the console defaults to your selection.
-. In the Language section, select *Default browser language* to use the default browser language settings. Otherwise, select the language that you want to use for the console.
+. In the *Language* section, select *Default browser language* to use the default browser language settings. Otherwise, select the language that you want to use for the console.
+. In the *Applications* section:
+.. You can view the default *Resource type*. For example, if the {ServerlessOperatorName} is installed, the default resource type is *Serverless Deployment*. Otherwise, the default resource type is *Deployment*.
+.. You can select another resource type to be the default resource type from the *Resource Type* field. 
```

Comment:
not sure if it's worth calling out DeploymentConfig and Last used explicitly as there are just two more apart from serverless and Deployment

<img width="918" alt="image" src="https://user-images.githubusercontent.com/5129024/187366826-64705fdd-967e-476f-9258-132f9d075be2.png">

Reply:
@invincibleJai IMHO, it is intuitive from Line 22 for the user to select the resource type as the default resource type from the available options in the **Resource type** field. I believe we don't need to call out them explicitly.
Please let me know if you have other thoughts. Thank you!

Cc @divyanshiGupta @sanketpathak